__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Please dont ask me to scroll to another part of the notebook to find the things which you are meant to be showing in this section.</p><p>Can you combine all of your results into a single graph (do not plot accuracy against model number).</p><p> </p><p>Why didnt you remove these outliers and see if that was the problem? If you didn't, then you can't claim that was the problem with certainty.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I'll be moving this to the same section.</p><p> </p><p>I'm not certain the reason of the problem was the outliers, I'm probably incorrect and will look at it closely asap.</p>